### ISSUE TITLE
HADS modes

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#1033]
There have been multiple ways of using HADS in the show
There might be options to:

- stay in the vortex (old behavior)
- fly elsewhere (current behaviour)
- remat after some time 
- remat after the entity that caused the damage is dead

We could also make HADS activate when there's shooting or explosions nearby, not just directed at the TARDIS.
Another option is hostile action DISPERSAL system from s9

Basically, there could be many options in one with the HADS control
